Client/Frontend/UserContent/UserScripts/CookieControl.js

```diff
@@ -39,3 +39,24 @@ if (Object.getOwnPropertyDescriptor(window, 'sessionStorage')) {
         },
     });
 }
+
+(() => {
+  // Access to caches should be denied when user has blocked all Cookies.
+  const makeFailingPromiseFunction = () => {
```

Comment:
We may want to anonymize this function `.toString` before returning it.

Not-blocking